randovania/gui/ui_files/preset_generation.ui

```diff
@@ -159,7 +159,7 @@
            <item>
             <widget class="QLabel" name="logical_pickup_placement_description">
              <property name="text">
-              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This setting controls what pickups Randovania places logically. Any pickups placed in this way are guaranteed to be obtainable. Other pickups may not be required to finish or even be possible to get.&lt;/p&gt;&lt;p&gt;Minimal: Only the pickups required to beat the game are placed logically. This is the default setting.&lt;/p&gt;&lt;p&gt;Major pickups: Every major pickup in the preset is placed logically.&lt;/p&gt;&lt;p&gt;All pickups: Every pickup, including ammo and expansions, is placed logically.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Warning:&lt;/span&gt; Selecting 'Major pickups' or 'All pickups' will make generation fail more frequently.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This setting controls what pickups are guaranteed to be obtainable. Other pickups may not be required to finish or even be possible to get.&lt;/p&gt;&lt;p&gt;Minimal: Only the pickups required to beat the game are guaranteed to be obtainable. This is the default setting.&lt;/p&gt;&lt;p&gt;Major pickups: Every major pickup in the preset is guaranteed to be obtainable.&lt;/p&gt;&lt;p&gt;All pickups: Every pickup, including ammo and expansions, is guaranteed to be obtainable.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Warning:&lt;/span&gt; Selecting 'Major pickups' or 'All pickups' will make generation fail more frequently.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
I'd omit "Other pickups may not be required to finish or even be possible to get" personally

Reply:
agreed

Reply:
also, I really think this should update dynamically as you select a mode, like DLR mode does. take a look at the dock rando tab as a reference for what I mean

Reply:
also also I'm not sure we need to include "this is the default setting"